tasks/kmt.py

```diff
@@ -969,17 +977,21 @@ def build(
 
     assert len(domains) > 0, f"no vms found from list {vms}. Run `inv -e kmt.status` to see all VMs in current stack"
 
+    if component == "security-agent":  # security-agent build does not build ebpf files by default
```

Comment:
Maybe we can run this every time, and pass `--skip-object-files` to `build`. It would make changes easier to test by reducing the possible paths.

Reply:
Done!